tests/integration/components/polaris-setting-toggle-test.js

```diff
@@ -160,3 +160,41 @@ test('it handles the suppied action correctly', function(assert) {
   click(settingActionButtonSelector);
   assert.ok(this.get('actionFired'), 'fires the action when button clicked');
 });
+
+test("it obeys the passed-in action hash's loading and disabled flags", function(assert) {
+  this.setProperties({
+    isLoading: true,
+    isDisabled: false,
+  });
+
+  this.render(hbs`
+    {{polaris-setting-toggle
+      action=(hash
+        text="Toggle"
+        loading=isLoading
+        disabled=isDisabled
+        onAction=(action (mut dummy))
+      )
+    }}
+  `);
+
+  let button = find(settingActionButtonSelector);
+  assert.ok(
```

Comment:
this could also be written as 
```js
assert.dom(settingActionButtonSelector).hasClass('Polaris-Button--loading', 'button is in loading state when loading is true');
```

Reply:
It could... but I want to get this out quickly so wasn't going to update the test file to use the newer things 🤷‍♂️